changelog/7399.deprecation.rst

```diff
@@ -0,0 +1 @@
+Moved sunpy.physics.differential_rotation.diff_rot to sunpy.sun.models.diff_rot and added tests to sunpy.sun.tests.test_models. deprecated sunpy.physics.differential_rotation.diff_rot
```

Comment:
This changelog needs to be more user facing, items like the tests moving isn't useful information for users.